backend/app/GraphQL/Mutations/UpdateTeamDashboard.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\Team;
+use Illuminate\Support\Facades\Auth;
+
+final class UpdateTeamDashboard
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        $user = Auth::user();
+        $content = Team::find($args['id']);
+
+        if ($user['id'] === $content['teamLeader']['id']) {
```

Comment:
You can use `Auth::id()` on line 19 then you can delete `$user = Auth::user();` na